webpack/components/Table/__test__/useBulkSelect.test.js

```diff
@@ -0,0 +1,89 @@
+import { act, renderHook } from '@testing-library/react-hooks';
+import { useBulkSelect } from '../TableHooks';
+
+const isSelectable = () => true;
+const idColumn = 'errata_id';
+const metadata = {
+  error: null, selectable: 2, subtotal: 2, total: 2,
+};
+const results = [
+  {
+    errata_id: 'RHSA-2022:2031',
+    id: 311,
+    severity: 'Low',
+  },
+  {
+    errata_id: 'RHSA-2022:2110',
+    id: 17,
+    severity: 'Important',
+  },
+];
+
+it('returns a scoped search string based on inclusionSet', () => {
+  const { result } = renderHook(() => useBulkSelect({
+    results,
+    metadata,
+    idColumn,
+    isSelectable,
+  }));
+
+  act(() => {
+    result.current.selectOne(true, 'RHSA-2022:2031');
+  });
+
+  expect(result.current.fetchBulkParams()).toBe('errata_id ^ (RHSA-2022:2031)');
+});
+
+it('returns a scoped search string based on exclusionSet', () => {
+  const { result } = renderHook(() => useBulkSelect({
+    results,
+    metadata,
+    idColumn,
+    isSelectable,
+  }));
+
+  act(() => {
+    result.current.selectAll(true);
+  });
+
+  act(() => {
+    result.current.selectOne(false, 'RHSA-2022:2031');
+  });
+
+  expect(result.current.fetchBulkParams()).toBe('errata_id !^ (RHSA-2022:2031)');
+});
+
+it('adds search query to scoped search string based on exclusionSet', () => {
+  const { result } = renderHook(() => useBulkSelect({
+    results,
+    metadata,
+    idColumn,
+    isSelectable,
+  }));
+
+  act(() => {
+    result.current.selectAll(true);
+  });
+
+  act(() => {
+    result.current.updateSearchQuery('errata_id=RHSA-2022:2031');
+  });
+
+  expect(result.current.fetchBulkParams()).toBe('errata_id=RHSA-2022:2031');
```

Comment:
I think this test and the test below would make a bit more sense to the reader if it first had a non-blank exclusion set to add the query to. As it is now, it starts with nothing, so it doesn't really illustrate the "adding" part.

Reply:
`exclusionSet` is checked with when `selectAllMode` is `true`. 
I can add item to `exclusionSet` if you prefer an non-empty `exclusionSet`.

Reply:
Yeah, I'd prefer to see the expectation be "xxx and xxx" instead of just "xxx"